numba/entrypoints.py

```diff
@@ -19,6 +19,6 @@ def init_all():
     _already_initialized = True
 
     for entry_point in iter_entry_points('numba_extensions', 'init'):
-        logger.debug('Loading extension:', entry_point)
+        logger.debug('Loading extension:' + str(entry_point))
```

Comment:
```suggestion
        logger.debug('Loading extension: %s', entry_point)
```
It should be using the format string so that the `str()` is only executed if logging is enabled.